misc/cf-support

```diff
@@ -150,7 +150,8 @@ mkdir -p "$tmpdir"
 
 echo "Analyzing CFEngine core dumps"
 _core_log="$tmpdir"/core-dump.log
-if command -v coredumpctl >/dev/null; then
+if expr "$_sysctl_kernel_core_pattern" : ".*/systemd-coredump.*" > /dev/null 2>&1; then
```

Comment:
Did you forget to include a _sysctl_kernel_core_pattern variable somewhere?

Reply:
Seems I did. fixed, squashed, rebased pushed.